frontend/src/components/main/About.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+
+class About extends Component {
+    render() {
+        return(
+            <div className="container-fluid" id="about">
+                <div className="row">
+                    <div className="col-12 col-md-6">
+                        <h3>
+                            About Project MKB
+                        </h3>
+                        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Aperiam quaerat maiores recusandae excepturi, eius temporibus totam odit obcaecati autem impedit eos dolorum reiciendis eum consectetur, fuga sed facilis in exercitationem.</p>
+                    </div>
+                    <div className="col-12 col-md-6">
+                        <img src="https://via.placeholder.com/300" alt="" className="img-fluid"/>
```

Comment:
I noticed that this link doesn't work on my end.
when I first load landing page which contains this About component, it keeps loading and this cause another issue.
could you check if this place holder works fine on your end?

I checked below two place holders and worked fine
https://place-hold.it/300
https://picsum.photos/300